src/platforms/stm32/src/lib/sys.c

```diff
@@ -26,6 +26,9 @@
 
 #include "gpiodriver.h"
 
+void sys_tick_handler(void);
```

Comment:
Same here about `(void)` when there are no parameters.

Reply:
This is really a mystery even with Paul's addition of:
`target_compile_features(${PROJECT_EXECUTABLE} PUBLIC c_std_11)`
to src/platforms/stm32/src/CMakeLists.txt in PR #400, C std11 does not seem to be honored. I even tried adding `-std=c11` compiler flag to src/platforms/stm32/cmake/compile-flags.cmake, and with `cmake -DLOG_VERBOSE=true` I could see that that the `-std=c11` flag was actually stripped out of the final build flags. But this must be the reason that arm-none-eabi-gcc complains that there is no prototype for these functions.